pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -2631,7 +2632,7 @@ def test_onOffset(self, case):
     apply_cases = []
     apply_cases.append(
         (
-            CDay(),
+            1 * CDay(),
```

Comment:
What is the multiplication for? Same comment below a few times

Reply:
This keeps the type consistent through out the whole test. Lines with `2 * CDay()` etc, get detected as List[int, datetime], while the one without multiplication got detected as `CustomBusinessDay` iirc. I decided that since this works with the other tests, it should be safe to make this change. I agree that this is a kind of a dirty fix, so if there is a better way, I'll make corresponding edits.

Reply:
So this fails for `apply_cases` then? If so should just annotate that list